library/find_unused_disk.py

```diff
@@ -27,7 +32,9 @@
   tasks:
     - name: run module
       find_unused_disk:
+        disk_size: '10g'
```

Comment:
It would be nice if it were called `min_size` since that how it gets interpreted.